os/arch/Kconfig

```diff
@@ -454,6 +454,19 @@ config ARCH_IRQPRIO
 		interrupts with certain configuration options.  So this selection
```

Comment:
Minor: commit title says - 
"This commit picks these changes to arnv7a" -> "armv7a" please